Deployment/Frontend/index.js

```diff
@@ -0,0 +1,122 @@
+const identityProviders = [
+    {
+        ID: 'sample',
+        Display: 'sample',
+        ImageURL: ''
```

Comment:
You can add a default placeholder image to simulate a providers logo :)
```suggestion
        ImageURL: 'ImageURL: 'https://via.placeholder.com/60''
```

Reply:
That's a good idea 👍